bokeh/models/__init__.py

```diff
@@ -32,8 +32,7 @@
 # Bokeh imports
 from ..core.property.dataspec import expr, field, value  # Legacy API
 from ..model import Model
-from .annotations import *
-from .arrow_heads import *
+from .annotations import *  # lgtm [py/polluting-import]
```

Comment:
I guess this is due to the lack of explicit `__all__` in `annotations.py`?

Reply:
Yes, I copied this from a later line importing widgets.